Library/include/CSP/Systems/ECommerce/ECommerceSystem.h

```diff
@@ -66,15 +66,21 @@ class CSP_API ECommerceSystem : public SystemBase
 	/// @brief Get product information from a shopify store within a space
 	/// @param SpaceId csp::common::String : space id of product
 	/// @param ProductId csp::common::String : Product id of product
-	/// @param Callback NullResultCallback : callback when asynchronous task finishes
-
+	/// @param Callback ProductInfoResultCallback : callback when asynchronous task finishes
 	CSP_ASYNC_RESULT void GetProductInformation(const common::String& SpaceId, const common::String& ProductId, ProductInfoResultCallback Callback);
 
+	/// @brief Get product information from a shopify store within a space, by providing a VariantId
```

Comment:
/// @brief Get product information from a shopify store within a space, by providing an array of VariantIds